### PR TITLE
improving buttons

### PR DESCRIPTION
**Pull request by kumarishalini6**
#155 
I have improve style of every buttons in the forms.
InkedReact App - Personal - Microsoft​ Edge 5_21_2021 09_45_27 PM_LI

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -46,10 +46,6 @@ function App() {
               path="/patients"
               component={Register}
             />
-            <Route
-              path="/appointment"
```
Comment (chetan-rns) [inline src/App.js:50]: Any specific reason why `appointment` is removed from the navbar?

File: src/components/PatientRegister.css [hunk 2]
```diff
@@ -25,3 +25,36 @@
 .container {
     text-align: 'center';
 }
+ 
+.button {
+    position: relative;
+    /* float: right; */
+    margin-top: 20px;
+    /* margin-right: 10%; */
```
Comment (chetan-rns) [inline src/components/PatientRegister.css:33]: Can we remove these commented lines? Everything else looks good to me :+1:

**Review by chetan-rns — CHANGES_REQUESTED** [review]

File: src/components/PatientRegister.css (hunk 2 above)
Comment (kumarishalini6) [reply src/components/PatientRegister.css:33]: Just give me 5min

**Comment by kumarishalini6** [comment]
done

File: src/App.js (hunk 1 above)
Comment (kumarishalini6) [reply src/App.js:50]: because of that line the appointment buttons goes to the patient page. it was 2 times, that's why i have deleted the dummy one.

Comment (chetan-rns) [reply src/App.js:50]: @kumarishalini6  ^^

Comment (chetan-rns) [reply src/App.js:50]: :+1:

**Comment by chetan-rns** [comment]
@kumarishalini6 Also please make sure to elaborate the commit messages in the future PRs. It would help others to understand the change introduced.

**Review by chetan-rns — APPROVED** [review]

**Comment by kumarishalini6** [comment]
Sure

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**